.scripts/check_app_name.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+shopt -s lastpipe   # Run last command in a pipeline in the current shell.
+
+# Colors
+LIGHTCYAN='\033[1;36m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'    # No Color
+
+CANONICAL_TITLE="Catima"
+# From "ast" is empty file list.
+ALLOWLIST=("ar" "bn" "fa" "hi" "kn" "ml" "mr" "ta" "zh-rTW" "ast" "bn-rIN" "bs" "cy" "he-rIL" "is" "iw" "lb" "oc")
+
+success=1
+
+find ../app/src/main/res/values* -name "strings.xml" | while read xml; do
+    LANG=$(echo "$xml" | sed -n 's|.*/values-\([^/]*\)/strings.xml|\1|p')
+    LANG=${LANG:-en}
+
+    APP_NAME=$(grep -oP '<string name="app_name">\K[^<]+' "$xml" | head -n1)
+
+    if [[ " ${ALLOWLIST[@]} " =~ " ${LANG} " ]]; then
+        continue
+
+    elif [[ ! -s "$xml" || -z "$APP_NAME" ]]; then
+        echo -e "${RED}Error: ${LIGHTCYAN}app_name is missing or file is empty in $xml ($LANG)."
```

Comment:
I guess it doesn't matter in the code as it is now, but it would make future bugs more likely to happen, so let's make sure to add ${NC} to the end here so each statement does properly reset the colour.